### ISSUE TITLE
Many NAs in the gene-wise Fst file

### ISSUE THREAD

**Issue by 473021677** [EnvGen/POGENOM#19]
Hi,
   I am yuanyang, a post doctorate. I have run the pogenom.pl and the resulting files have been generated. However, there are many NA in the gene-wise Fst file. I am not sure what is wrong with it. I really need your help. I have appended the gene-wise Fst file. If you could help, I really appreciate it. Thanks.

Best regards,
Yang Yuan
[Uploading j46-pa_bin.15.fst-per-gene.txt…]()

**Comment by lfdelzam** [comment]
Have you used input Pogenom?

**Comment by 473021677** [comment]
Yes, I have used the Input_POGENOM pipeline. I am not sure whether so many NAs in the gene-wise Fst file might be due to the insufficient coverage of these genes in either or both of the two samples compared.

Best regards,
Yang Yuan
